src/app/view/perspectives/post-it/document/post-it-document.component.scss

```diff
@@ -76,6 +83,7 @@
     padding: 0 3px;
     width: 100%;
     pointer-events: none;
+    color: #000000;
```

Comment:
I am not sure if we really want to use pure black color anywhere in our application.

Reply:
We do. It's the default for text in inputs.